server/server.go

```diff
@@ -454,20 +454,21 @@ func cutPluginCommandF(w http.ResponseWriter, slashCommand *MMSlashCommand, tag,
 		WriteErrorResponse(w, NewError(err.Error(), nil))
 		return nil
 	}
-
-	msg := fmt.Sprintf("Tag %s created. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", tag)
+	
+	cmdFlags := fmt.Sprintf("Flags: tag `%s`, repo `%s`, commitSHA `%s`, force `%v`.", tag, repo, commitSHA, force)
+	msg := fmt.Sprintf("%s\nTag created. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", cmdFlags)
 	if err := createTag(ctx, client, Cfg.GithubOrg, repo, tag, commitSHA); errors.Is(err, ErrTagExists) {
 		if !force {
-			WriteErrorResponse(w, NewError(fmt.Errorf("Tag %s already exists, not generating any artifacts. Use --force to regenerate artifacts.", tag).Error(), nil))
+			WriteErrorResponse(w, NewError(fmt.Errorf("%s\nTag already exists. Not generating any artifacts. Use --force to regenerate artifacts. ", cmdFlags).Error(), nil))
 			return nil
 		}
-		msg = fmt.Sprintf("Tag %s exists. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", tag)
+		msg = fmt.Sprintf("%s\nTag exists. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", cmdFlags)
 	} else if err != nil {
 		WriteErrorResponse(w, NewError(err.Error(), nil))
 		return nil
 	}
 
-	WriteEnrichedResponse(w, "Plugin Release Process", msg, "#0060aa", IN_CHANNEL)
+	WriteEnrichedResponse(w, fmt.Sprintf("%s\nPlugin Release Process", cmdFlags), msg, "#0060aa", IN_CHANNEL)
```

Comment:
I think this will double include the cmd flags, lets remove it from here

```suggestion
	WriteEnrichedResponse(w, "Plugin Release Process", msg, "#0060aa", IN_CHANNEL)
```

Reply:
Ohh, I missed it. Sorry for that :/